cms/static/js/views/pages/container.js

```diff
@@ -17,6 +17,7 @@ define(['jquery', 'underscore', 'gettext', 'js/views/pages/base_page', 'common/j
                 'click .edit-button': 'editXBlock',
                 'click .visibility-button': 'editVisibilitySettings',
                 'click .duplicate-button': 'duplicateXBlock',
+                'click .move-button': 'moveXblock',
```

Comment:
Nit: I prefer ``moveXBlock`` and that is the naming convention used throughout this file.